### DISCUSSION TITLE
只有我觉得model的弹出动画很别扭吗？

### DISCUSSION THREAD

**Discussion by zeni18** [ant-design/ant-design#36879] [General]
### Reproduction link

[https://ant.design/components/modal-cn/](https://ant.design/components/modal-cn/)

### Steps to reproduce

1

### What is expected?

实现淡出淡入的动画

### What is actually happening?

从点击位置逐渐放大

| Environment | Info |
| --- | --- |
| antd | undefined |
| React | 16.0 |
| System | macos |
| Browser | chrome |

---
只有我觉得model的弹出动画很别扭吗？能够实现从中间淡出淡入的效果，很多ui库都是这样的

**Comment by zhou94** [comment]
可以通过transitionName和maskTransitionName自定义实现
